metadata-ingestion/src/datahub/ingestion/source/mlflow.py

```diff
@@ -21,34 +26,84 @@
 from datahub.ingestion.api.source import Source, SourceCapability, SourceReport
 from datahub.ingestion.api.workunit import MetadataWorkUnit
 from datahub.metadata.schema_classes import (
+    AuditStampClass,
+    BrowsePathsV2Class,
+    ContainerClass,
+    ContainerPropertiesClass,
+    DataPlatformInstanceClass,
+    DataProcessInstanceOutputClass,
+    DataProcessInstancePropertiesClass,
+    DataProcessInstanceRunEventClass,
+    DataProcessInstanceRunResultClass,
+    DataProcessRunStatusClass,
     GlobalTagsClass,
     MLHyperParamClass,
     MLMetricClass,
     MLModelGroupPropertiesClass,
     MLModelPropertiesClass,
+    MLTrainingRunPropertiesClass,
+    SubTypesClass,
     TagAssociationClass,
     TagPropertiesClass,
+    TimeStampClass,
+    VersionPropertiesClass,
+    VersionSetKeyClass,
+    VersionSetPropertiesClass,
     VersionTagClass,
-    _Aspect,
+)
+from datahub.metadata.urns import (
+    DataPlatformUrn,
+    VersionSetUrn,
 )
 
 T = TypeVar("T")
 
 
+class ContainerKeyWithId(ContainerKey):
+    id: str
+
+
+@dataclass
+class Container:
```

Comment:
Should we define this somewhere else?

Reply:
we can now start using the datahub.sdk.container -> Container class

Reply:
done